### ISSUE TITLE
Design a logging, secure audit, and forensic architecture.

### ISSUE THREAD

**Issue by rasputtintin** [mojaloop/project#2038]
## **Goal**:
`As a` Security Architect

`I want to adapt the current architecture to provide immutable logging and audit capabilities
  
`so that I can provide a reliable log and audit trail that can be used as a basis to perform forensics.

**Acceptance Criteria**:
- [ ] TBD

**Complexity**: <Medium

**Uncertainty**: Low

---

**Tasks**:
- [ ] TBD [ @? ]

**Done**
- [ ] Acceptance Criteria pass
- [ ] Designs are up-to date
- [ ] Unit Tests pass
- [ ] Integration Tests pass
- [ ] Code Style & Coverage meets standards
- [ ] Changes made to config (default.json) are broadcast to team and follow-up tasks added to update helm charts and other deployment config.
- [ ] TBD

## **Pull Requests**:
- [ ] TBD

## **Follow-up**:
- N/A

**Dependencies**:
- N/A

## **Accountability**:
- Owner: @rasputtintin @pedrosousabarreto @dpcMomo 
- QA/Review: @godfreykutumela

**Comment by godfreykutumela** [comment]
Blocked by Define business/functional requirements for logging, audit and forensic issue

**Comment by godfreykutumela** [comment]
Merged into the reference architecture
